HyperV/scripts/generateCinderCfg.ps1

```diff
@@ -0,0 +1,64 @@
+Param(
+    [Parameter(Mandatory=$true)][string]$configDir,
+    [Parameter(Mandatory=$true)][string]$templatePath,
+    [Parameter(Mandatory=$true)][string]$serverIP,
+    [Parameter(Mandatory=$true)][string]$rabbitUser,
+    [Parameter(Mandatory=$true)][string]$logDir,
+    [Parameter(Mandatory=$true)][string]$lockPath
+)
+
+
+$username = 'Administrator'
```

Comment:
This looks a bit scary :) Since we already run the openstack services with an account having access to the share, I guess we could skip this.